src/middleware/withAPIPageLogger.ts

```diff
@@ -0,0 +1,56 @@
+import { createLogger } from "@fewlines/fwl-logging";
+import { NextApiRequest, NextApiResponse } from "next";
+import { Handler } from "src/pages/api/identity";
+
+type WithAPIPageLogger = (
+  request: NextApiRequest,
+  response: NextApiResponse,
+) => Promise<JSON | void>;
+
+export function withAPIPageLogger(handler: Handler): WithAPIPageLogger {
+  const logger = createLogger("connect-account", "json").withMeta({
+    process: "getServerSideProps",
+  });
+
+  const processTimeStart = process.hrtime.bigint();
+
+  return async (request, response) => {
+    handler(request, response)
+      .then((data) => {
+        const processTimeEnd = process.hrtime.bigint();
+
+        logger.log("Success", {
+          method: request.method,
+          statusCode: request.statusCode,
+          path: request.url,
+          duration: (
+            (processTimeEnd - processTimeStart) /
+            BigInt(1000)
+          ).toString(),
```

Comment:
We didn't need `.toString` in FWL, is it needed there?

Reply:
We do, otherwise I get and error saying smth along the line of `Do not know how to serialize a BigInt` (I can reproduce the error if you want). We just need to add in the `log` function a type check like `typeof value === 'bigint' ? value.toString(): value`. I can write a task about it if you want.

Reply:
I'm curious as to why there is no bug in FWL, is there and we just don't know it?

Reply:
There are no tests that try to log a `bigInt` value. New task I guess ?

Reply:
ah yes, I had to fix it in FWL too ^^', my bad